users/woodsdou/index.md

```diff
@@ -3,64 +3,58 @@ title: Doug-personal
 Doug Woods
-================================
+==============
+woodsdou@oregonstate.edu
 
-https://rtrp.github.io/osu-transport/woodsdou/
+https://rtrp.github.io/osu-transport/users/woodsdou/
 
-I am beginning my 2nd year as a PhD candidate in Nuclear Engineering in the [School of Nuclear Science and Engineering](https://ne.oregonstate.edu) at [Oregon State University](https://oregonstate.edu). My major advisor is [Todd Palmer](https://rtrp.github.io/osu-transport/palmerts/).
-
-
-## Education
-
-* Ph.D. in Nuclear Engineering at Oregon State University, Candidate
-* M.S. in Nuclear Engineering[^1] at Oregon State University, February 2016
-* B.S. in Nuclear Engineering at Oregon State University, March 2013
+I am a Ph.D. candidate in Nuclear Engineering in the [School of Nuclear Science and Engineering](https://ne.oregonstate.edu) at [Oregon State University](https://oregonstate.edu). My major advisor is [Todd Palmer](https://rtrp.github.io/osu-transport/palmerts/). I am also a licenced reactor operator for the [Oregon State TRIGA Reactor](http://radiationcenter.oregonstate.edu/oregon-state-triga-reactor-0).
 
 ***
 
 [https://web.engr.oregonstate.edu/~woodsdou/wiki/HomePage](https://web.engr.oregonstate.edu/~woodsdou/wiki/HomePage)
 
 ***
 
-## Current Research
+## Research
 
-High-order finite element radiation transport using meshes with curved surfaces using the open source finite element library [MFEM](https://mfem.org).
+We have demonstrated the feasibility of high-order finite element radiation transport using meshes with curved surfaces using the open source finite element library [MFEM](https://mfem.org). The image below[^1] illustrates a mesh with 3<sup>rd</sup> order polynomial curved surfaces.
 
-***
+<img src="https://github.com/rtrp/osu-transport/blob/woods-edits/users/woodsdou/images/AdamsDiff2DwMeshBlue.png" width="450">
 
-## Current projects
+With mesh refinement and/or increasing the finite element order, we see solutions converging to the analytic solution. The data points on the following image[^1] show errors between the DGFEM transport solution and the analytic MMS solution. Lines connect data points calculated using the same finite element order.
 
-* Dissertation
-  * Currently doing literature review and collecting material for writing my dissertation proposal
+<img src="https://github.com/rtrp/osu-transport/blob/woods-edits/users/woodsdou/images/plotConvergenceRates_15.png" width="400">
 
+Diffusion limit calbulations can exhibit unphyiscial oscillations in the boundadry layer solution. If the solution is near zero, the oscillations will cause the solution to drop below zero as seen by white space in the image below[^1]. In the context of thermal radiation transport, negative fluxes mean negative temperatures, which can lead to the equations of state to calculate negative densities and pressures.
```

Comment:
"negative fluxes mean negative temperatures, which can lead to the equations of state to calculate negative densities and pressures."

that sounds... 💩 

....yes I'm getting sick of reading papers.